docs/tutorial/createdb.rst

```diff
@@ -0,0 +1,112 @@
+.. _ref_tutorial_createdb:
+
+Database and Schema
+===================
+
+First step in a brand new project is to create the database for it:
+
+.. code-block:: edgeql-repl
+
+    db> CREATE DATABASE tutorial;
+    CREATE
+
+The above :ref:`command <ref_eql_ddl_databases>` creates a new
+:ref:`database <ref_datamodel_databases>` in the EdgeDB instance. Now
+we should connect to it:
+
+.. FIXME "\c" currently causes lexer errors in doc tests
+
+.. code-block::
+
+    db> \c tutorial
+    tutorial>
+
+Now we need to set up the schema. Let's set up a basic schema for a
+movies database. It will have 2 objects: movies and people who directed and
+acted in them.
+
+For the next step, there are two ways of setting this up. Please pick
+one of the methods: :ref:`ref_tutorial_createdb_sdl` (recommended) or
+:ref:`ref_tutorial_createdb_ddl`.
+
+
+.. _ref_tutorial_createdb_sdl:
+
+SDL
+---
+
+The :ref:`EdgeDB schema definition language <ref_eql_sdl>` provides a
+way to describe a :ref:`migration <ref_eql_ddl_migrations>` to a
+specific schema state. It is great for setting up a new database because it
+focuses on expressing the final :ref:`types <ref_eql_sdl_object_types>` and
+their :ref:`relationships <ref_eql_sdl_links>` without worrying about
+the order of the definitions.
+
+Migrations have to be done inside a :ref:`transaction
+<ref_eql_statements_start_tx>`:
+
+.. code-block:: edgeql-repl
+
+    tutorial> START TRANSACTION;
+    START TRANSACTION
+    tutorial> CREATE MIGRATION movies TO {
+    .........     type Movie {
+    .........         required property title -> str;
+    .........         # the year of release
+    .........         property year -> int64;
+    .........         required link director -> Person;
+    .........         required multi link cast -> Person;
+    .........     }
+    .........     type Person {
+    .........         required property first_name -> str;
+    .........         required property last_name -> str;
+    .........     }
+    ......... };
+    CREATE MIGRATION
+    tutorial> COMMIT MIGRATION movies;
+    COMMIT MIGRATION
+    tutorial> COMMIT;
+    COMMIT TRANSACTION
+
+The name of a migration doesn't matter much beyond providing a way to
+specify the particular migration must be committed. Once the
+transaction is committed the schema is updated and we're ready to
+:ref:`populate the database with data <ref_tutorial_queries>`.
+
+
+.. _ref_tutorial_createdb_ddl:
+
+DDL
+---
+
+The :ref:`data definition language <ref_eql_ddl>` focuses on
```

Comment:
Please add a separate paragraph saying that the entire DDL section is an alternative approach to SDL and need not be followed if the reader has completed the steps in the previous section. Make it in bold font.

Reply:
OK